transmitterIO.js

```diff
@@ -104,6 +103,18 @@ module.exports = async (options, storage, storageLock, client, fakeMeter) => {
     });
   };
 
+  const removeBTDevices = () => {
+    const btName = `Dexcom${txId.slice(-2)}`;
+
+    removeBTDevice(btName);
+
+    if (txAddress) {
+      const btAddressName = txAddress.split(':').join('-').toUpperCase();
```

Comment:
Why did you do a split and join here? I'm trying to mimic this in Logger and need to know the format that is reported by xdrip-js in txAddress and the string format required for "mac" in bt-device -r <mac>.